go.mod

```diff
@@ -54,44 +51,105 @@ require (
+	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20190724013045-ca1201d0de80 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20190923162816-aa69164e4478 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200222125558-5a598a2470a0 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200226121028-0de0cce0169b => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200301022130-244492dfa37a => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200501053045-e0ff5e5a1de5 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200506145744-7e3656a0809f => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200513185701-a91f0712d120 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200520182314-0ba52f642ac2 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200625001655-4c5254603344 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20200822124328-c89045814202 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20201021035429-f5854403a974 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20210525063256-abc453219eb5 => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f => golang.org/x/net v0.7.0
+	golang.org/x/net v0.0.0-20220722155237-a158d28d115b => golang.org/x/net v0.7.0
+	// golang.org/x/text
+	golang.org/x/text v0.0.0-20170915032832-14c0d48ead0 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.0 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.2 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.3 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.4 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.5 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.6 => golang.org/x/text v0.3.8
+	golang.org/x/text v0.3.7 => golang.org/x/text v0.3.8
```

Comment:
why do we have to maintain such a complex replacement array?

Reply:
Hi @ytsarev , I don't like it much either. I tried to bump core modules to avoid using old modules in module tree. Unfortunately they are there. It's not a lot of libraries, rather they are used differently across different dependencies.

For example, if I don't do a replace in `go.mod`, I'll see this in our dependency tree (`go.sum`)
```
golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
golang.org/x/text v0.3.0/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
golang.org/x/text v0.3.1-0.20180807135948-17ff2d5776d2/go.mod h1:NqM8EUOU14njkJ3fqMW+pc6Ldnwhi/IjpwHt7yyuwOQ=
golang.org/x/text v0.3.2/go.mod h1:bEr9sfX3Q8Zfm5fL9x+3itogRgK3+ptLWKqgva+5dAk=
golang.org/x/text v0.3.3/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
golang.org/x/text v0.3.4/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
golang.org/x/text v0.3.5/go.mod h1:5Zoc/QRtKVWzQhOtBMvqHzDpF6irO9z98xDceosuGiQ=
golang.org/x/text v0.3.7/go.mod h1:u+2+/6zg+i71rQMx5EYifcz6MCKuco9NR6JIITiCfzQ=
golang.org/x/text v0.7.0 h1:4BRB4x83lYWy72KwLD/qYDuTu7q9PjSagHvijDw7cLo=
golang.org/x/text v0.7.0/go.mod h1:mrYo+phRRbMaCq/xk9113O4dZlRixOauAjOtrjsXDZ8=
```
but depndabot is disingenuous and will mark all `golang.org/x/text` that are older than `v.0.3.8` (see: [dependabot alert](https://github.com/k8gb-io/k8gb/security/dependabot/42))

---

I see that dependabot only marked `golang.org/x/net` in k8gb, the [rest was marked in terratests](https://github.com/k8gb-io/k8gb/security/dependabot) ( `crypto` and `golang.org/x/text`). Since the same problem is also in k8gb, I'll mark it there soon too. 


For now, I'm replacing only what is strictly necessary to avoid dependabot alerts.